examples/foreach-statement/foreach_statement.bal

```diff
@@ -1,29 +1,27 @@
 import ballerina/io;
 
 public function main() {
-    float v1 = sum([10.5, 20.5, 30.5]);
-    float v2 = sum2([10.5, 20.5, 30.5]);
-    io:println("v1:", v1, " v2:", v2);
-}
 
-function sum(float[] v) returns float {
-    float r = 0.0;
-    // `foreach` statement can be used to iterate an `array`.
-    // Each iteration returns an element in the `array`.
-    foreach float x in v {
-        r += x;
+    // Loop through a list
+    string[] names = ["Bob", "Jo", "Ann", "Tom"];
+    foreach string name in names {
+        io:println(name);
     }
 
-    return r;
-}
+    // Iterate a structure
+    map<int> grades = { Bob : 65, Jo : 70, Ann : 75, Tom : 60};
+    int sum = 0;
+    foreach int grade in grades {
+      sum += grade;
+    }
+
+    io:println("Average :", sum/grades.length());
 
-function sum2(float[] v) returns float {
-    float r = 0.0;
-    // `m ..< n` creates a value that when iterated over will give the
-    // integers starting from `m` that are `< n`.
-    foreach int i in 0 ..< v.length() {
-        r += v[i];
+    // binding patterns can be used with `foreach` statement
+    [string, int][] resultList = [["Bob", 65], ["Jo", 70], ["Ann", 75], ["Tom", 60]];
+    // binding pattern [string, int] [name, grade] used with the `foreach` statement
```

Comment:
Instead, shall we say that the values in resultList are assigned to name, grade variables in the binding pattern